protobind/node.go.template

```diff
@@ -21,6 +21,21 @@ import (
 	"go.uber.org/zap"
 )
 
+{{ if eq .Blockchain "flow" }}
+var (
+    // DefaultDiscoveryHintsSystemd default regular expression to detect Flow nodes run by systemd
+	DefaultDiscoveryHintsSystemd = "flow-(access|collection|verification|execution|consensus).service"
+
+    // DefaultDiscoveryHintsDocker default regular expression to detect Flow nodes run by docker
+	DefaultDiscoveryHintsDocker = "flow-go"
+)
+{{ else if eq .Blockchain "solana" }}
+var (
+	DefaultDiscoveryHintsSystemd = "solana-validator.service"
+	DefaultDiscoveryHintsDocker = "solana-validator"
```

Comment:
Should add comments here to avoid future linter errors in CI.

Reply:
Addressed in 264b24eec14d.